Makefile

```diff
@@ -1551,7 +1548,7 @@ unsafe-clean-c-deps:
 .PHONY: clean-execgen-files
 clean-execgen-files:
 	find ./pkg/sql/colexec -type f -name '*.eg.go' -exec rm {} +
-	find ./pkg/sql/exec -type f -name '*.eg.go' -exec rm {} + || true
+	test -d ./pkg/sql/exec && find ./pkg/sql/exec -type f -name '*.eg.go' -exec rm {} + || true
```

Comment:
tip: I'd have done `-exec rm -f ...` and removed `|| true` so that you'd still grab permission errors from `find` if any.